spec/GraphQLOverHTTP.md

```diff
@@ -500,9 +501,9 @@ execution - see "partial response" in the GraphQL specification.
 The server SHOULD NOT use a `4xx` or `5xx` status code.
 
 Note: For compatibility with legacy servers, this specification allows the use
-of `4xx` or `5xx` status code for failed requests using the `application/json`
-media type, but it is strongly discouraged. To use `4xx` and `5xx` status codes,
-please use the `application/graphql+json` media type.
+of `4xx` or `5xx` status code for failed requests where the responses uses the
```

Comment:
Consider `requests where the response uses` or `requests where the responses use`